mbuild/tests/test_compound.py

```diff
@@ -960,3 +960,27 @@ def test_from_pybel_monolayer(self):
         #            monolayer.unitcell.GetC()/10], 
         #        rtol=1e-3)
 
+    @pytest.mark.skipif(not has_matplotlib or not has_networkx, reason="Networkx and/or matplotlib not available")
+    def test_visualize_bond_graph_with_hexane(self, hexane):
+        import matplotlib
+        from matplotlib import pyplot as plt
+        import matplotlib.testing.compare as compare
```

Comment:
I think this line can be dropped?

Reply:
Yeah. Even accounting for randomness, `spring_layout` in the `networkx` package was not producing similar images. So, we can certainly remove `import matplotlib.testing.compare as compare`